CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 * **Perl**: Expanded Perl support to include Perl test files and template toolkit (\*.t, \*.tt).
   [pull request #148](https://github.com/refined-bitbucket/refined-bitbucket/pull/148).
+  
+# 3.7.0 (2018-02-16)
+
+### Improvements:
+
+* Bring old link color (blue) back to pull request and repository titles,
+[pull request #153](https://github.com/refined-bitbucket/refined-bitbucket/pull/153).
 
```

Comment:
This is not going to be released exactly today 😅 , remove lines 7-8, and locate the Features heading above the language support, it would end up looking like this:

# Next Release
 
 ### Features:
 
 * Bring old link color (blue) back to pull request and repository titles,
 [pull request #153](https://github.com/refined-bitbucket/refined-bitbucket/pull/153).

 ### Language support:
  
 * **Perl**: Expanded Perl support to include Perl test files and template toolkit (\*.t, \*.tt).
    [pull request #148](https://github.com/refined-bitbucket/refined-bitbucket/pull/148).